tools/fleet-slackbot/claude-client.test.js

```
@@ -73,12 +75,12 @@ test("runAgentLoop forces a final no-tools response when the budget is exhausted
   assert.equal(result, "Best-effort answer with partial data.");
   assert.equal(
     calls.length,
-    MAX_TOOL_CALLS + 1,
-    `expected ${MAX_TOOL_CALLS} tool-call turns plus one forced no-tools call`
+    TEST_MAX_TOOL_CALLS + 1,
+    `expected ${TEST_MAX_TOOL_CALLS} tool-call turns plus one forced no-tools call`
   );
 
   // All in-budget calls must NOT disable tool use.
-  for (let i = 0; i < MAX_TOOL_CALLS; i++) {
+  for (let i = 0; i < TEST_MAX_TOOL_CALLS; i++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)